### PR TITLE
Include Mongolid Library

### PR DESCRIPTION
**Pull request by ravanscafi**
Easy, powerful and ultrafast ODM for PHP7 build on top of the [new mongodb driver](https://docs.mongodb.org/ecosystem/drivers/php/).

Mongolid

### CODE REVIEW

File: source/drivers/php-libraries.txt [hunk 1]
```diff
@@ -60,6 +62,8 @@ Framework Integrations
     library and includes an abstract model, a helper class for handling bulk
     writes, database wrappers for Laravel and Lumen, and some necessary helper
     functions.
+
+  - `Mongolid Laravel <https://github.com/leroy-merlin-br/mongolid-laravel>`_: Easy, powerful and ultrafast MongoDB ODM for Laravel.
```
Comment (jmikola) [inline source/drivers/php-libraries.txt]:
```suggestion
  - `Mongolid Laravel <https://github.com/leroy-merlin-br/mongolid-laravel>`_:
    Laravel package for the MongoLid ODM. Includes integration for the auth
    and queue failed job components.
```

I don't think this needs to repeat the marketing description for the main library. It'd be more helpful to note what specific integrations it provides. AFAICT, this doesn't have anything to do with Eloquent (unlike the jenssegers library).

File: source/drivers/php-libraries.txt [hunk 2]
```diff
@@ -37,6 +37,8 @@ Stand-alone Libraries
   **mongodb** extension. It is schema-less and supports fast object hydration
   and persistence, which makes it well-suited for modeling aggregation results.
 
+- `Mongolid <https://github.com/leroy-merlin-br/mongolid>`_: Easy, powerful and ultrafast ODM for PHP and MongoDB.
```
Comment (jmikola) [inline source/drivers/php-libraries.txt]:
```suggestion
- `Mongolid <https://github.com/leroy-merlin-br/mongolid>`_: Easy, powerful,
   and ultrafast ODM for PHP and MongoDB. Implements both ActiveRecord and
   DataMapper design patterns.
```

Fixing word wrap, Oxford comma, and adding another sentence to distinguish this library from other entries.

**Comment by ravanscafi** [comment]
@jmikola thanks for the review, I've updated it and gave more context.

File: source/drivers/php-libraries.txt [hunk 3]
```diff
@@ -60,7 +65,13 @@ Framework Integrations
     library and includes an abstract model, a helper class for handling bulk
     writes, database wrappers for Laravel and Lumen, and some necessary helper
     functions.
-    
+
+  - `Mongolid Laravel <https://github.com/leroy-merlin-br/mongolid-laravel>`_: 
+  Built on top of `Mongolid`, it provides many integrations with the framework,
+  while leveraging some core features of it. You might find some similarities 
+  with `Eloquent`, but this library is not a replacement for it, as it tries 
+  to follow MongoDB concepts.
```
Comment (jmikola) [inline source/drivers/php-libraries.txt]: Suggested rephrase:

> Built on top of the Mongolid ODM, this library provides integrations with the framework and some of its core features. There are some similarities with Eloquent but this library is designed specifically with MongoDB concepts in mind.

File: source/drivers/php-libraries.txt [hunk 4]
```diff
@@ -37,6 +37,11 @@ Stand-alone Libraries
   **mongodb** extension. It is schema-less and supports fast object hydration
   and persistence, which makes it well-suited for modeling aggregation results.
 
+- `Mongolid <https://github.com/leroy-merlin-br/mongolid>`_: Easy, powerful,
+   and ultrafast ODM for PHP and MongoDB. Implements ActiveRecord design
+   pattern accounting for MongoDB concepts. It supports embedded and referenced
+   documents.
```
Comment (jmikola) [inline source/drivers/php-libraries.txt]: Suggested rephrase:

> Easy, powerful, and ultrafast ODM for PHP and MongoDB. Implements both ActiveRecord and DataMapper design    patterns and supports embedded and referenced documents.

**Comment by jmikola** [comment]
Sorry that we lost track of this. I've incorporated the suggested changes and opened https://github.com/mongodb/docs-ecosystem/pull/669, so hopefully that can be merged soon.

**Comment by MongoCaleb** [comment]
Closed as per @jmikola

**Pull request closed without merging**